docs/tutorial/beginner.rst

```diff
@@ -0,0 +1,572 @@
+
+Datasets
+--------
+
+.. code-block:: text
+
+    datasets:
+    - 
+        label: egfr
+        node_files: ["tps-egfr-prizes.txt"]
+        edge_files: ["phosphosite-irefindex13.0-uniprot.txt"]
+        other_files: []
+        data_dir: "input"
+    
+In the configuration file, datasets are defined under the datasets section. Each dataset you define will be run against all of the algorithms specified in the configuration file. Each dataset entry begins with a label, which uniquely identifies it throughout the SPRAS workflow and outputs. The dataset must include the following types of files:
+
+- node_files: Input files listing the “prizes” or important starting nodes ("sources" or "targets") for the algorithm
+- edge_files: Input interactome or network file that defines the relationships between nodes
+- other_files: A placefolder for potential need for future delevelopment (double check if this is required)
+- data_dir: The file path of the directory where the input dataset files are located
+
+.. This example shows a dataset named egfr that provides both a prize node file and a network edge file, both of which will be used by SPRAS when running the chosen algorithms.
+
+Reconstruction Settings
+-----------------------
+
+.. code-block:: text
+
+    reconstruction_settings:
+    locations:
+        reconstruction_dir: "output/basic"
+
+The reconstruction_settings section controls where results are stored. In the configuration file, you specify the output directory with reconstruction_dir, which tells SPRAS where to save the reconstructed networks (in this example, output/basic). When working with multiple configuration files, you can set different paths for reconstruction_dir to keep results separated. If not specified, all results will be saved to the default directory output/.
+
+Analysis
+--------
+
+.. code-block:: text
+
+    analysis:
+    summary:
+        include: true
+    cytoscape:
+        include: true
+
+
+SPRAS includes multiple downstream analyses that can be toggled on or off directly in the configuration file. When enabled, these analyses run for each dataset and provide summaries or visualizations of the results produced by all enabled algorithms.
+
+In this example:
+
+- summary computes statistics for each algorithm’s parameter combination output, generating a summary file for all reconstructed subnetworks for each dataset.
+- cytoscape creates a Cytoscape session file (.cys) containing all reconstructed subnetworks for each dataset, making it easy to upload and visualize them directly in Cytoscape.
+
+
+Step 3: Running SPRAS on a provided example dataset 
+====================================================
+
+3.1 Running SPRAS with the Beginner Config
```

Comment:
when my tester did this step, it took to a second to spin up. Should mention this.

Also tester did not know when it was done, add the 7 out 7 steps done look at logs part to show people that it has been completed

Reply:
Issue for slowness was because he was running on an armchip and the docker image was amd64